packages/expo-keep-awake/src/ExpoKeepAwake.web.ts

```diff
@@ -1 +1,36 @@
-export default {};
+import { KeepAwakeEventType } from './ExpoKeepAwake.types';
+
+export function isAvailable() {
```

Comment:
```suggestion
export function isAvailableAsync(): Promise<boolean> {
```

There should only be one `default export {}` with all "native" methods inside of it.